cdap-ui/package.json

```diff
@@ -36,19 +38,31 @@
     "@babel/preset-react": "^7.0.0-beta.53",
     "@babel/preset-stage-2": "^7.0.0-beta.53",
     "@babel/preset-typescript": "^7.0.0-beta.53",
+    "@storybook/addon-actions": "^3.4.10",
+    "@storybook/addon-info": "^3.4.10",
+    "@storybook/addon-links": "^3.4.10",
```

Comment:
I don't see this being used yet. Is it installed by default when you install Storybook?